src/main/index.js

```diff
@@ -89,15 +89,15 @@ export { downloadBlob } from './utils';
  * @property {boolean | ScaleBarSpec} scaleBar Indicates whether scalebar should be printed (and optionally its options).
  * @property {number} scale Scale denominator.
  * @property {string} projection EPSG projection code.
- * @property {boolean | string} northArrow North arrow position.
+ * @property {WidgetPosition} northArrow North arrow position; specify `true` for default position (top right).
  * @property {ProjectionDefinition} projectionDefinitions Projection definitions to be newly registered.
- * @property {string} attributions Position where the attributions should be printed
+ * @property {WidgetPosition} attributions Position where the attributions should be printed; specify `true` for default position (bottom right).
```

Comment:
Why not keep {boolean|WidgetPosition} here ? I believe it would be more readable / user friendly :)

Reply:
You're right, maybe `true` should not be part of the widget position type. The intent was that `true` meant "default position for this widget", but it's indeed not very explicit.